state/application.go

```diff
@@ -694,6 +694,12 @@ func (a *Application) removeOps(asserts bson.D, op *ForcedOperation) ([]txn.Op,
 	}
 	ops = append(ops, secretLabelOps...)
 
+	secretLabelOps, err = a.st.removeConsumerSecretLabelOps(a.ApplicationTag())
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	ops = append(ops, secretLabelOps...)
+
```

Comment:
This isn't needed - consumers are not apps but individual units

Reply:
operator consumers are all apps?